synth_tools/commands/agents.py

```diff
@@ -38,8 +39,16 @@ def get_agent(
     """
     api = ctx.find_object(APIs)
     for i in agent_ids:
+        try:
+            a = api.syn.agent(i)
+        except KentikAPIRequestError as e:
+            if e.response.status_code == 404:
+                typer.echo(f"Agent {i} does not exists")
```

Comment:
In both failure cases we should call `fail`, so that `synth_ctl` returns non-zero exit code. I do realize that we may not print results for all of the agents that actually are available, but I think it is more important to give the right signal that something was missing, especially for a tool parsing the output.
Also typo in the message: "exist"